src/api/libcellml/component.h

```diff
@@ -45,6 +45,27 @@ class LIBCELLML_EXPORT Component: public ComponentEntity
     void setSourceComponent(const ImportPtr &imp, const std::string &name);
 
     /**
+     * @brief Appends the argument to the math string for this component.
+     * Appends the @p mathString to the existing math string for this component.
+     * @param mathString The @c std::string to append for this component.
+     */
+    void appendMath(const std::string &mathString);
```

Comment:
Nitpicking (so safe to ignore): I believe it ought to be ```math``` rather than ```mathString``` (same for lines 49, 90, 91).